upgrade-notes/backend_20240710_151842.md

```diff
@@ -0,0 +1,69 @@
+#### {Admin can manage customer structure on B2B domains} ([#3261](https://github.com/shopsys/shopsys/pull/3261))
```

Comment:
can you remove the curly brackets from name?

Reply:
fixed https://github.com/shopsys/shopsys/pull/3261/commits/45cb01b1c3bba631d5246fa156709375a91058df